### PR TITLE
Changed the fetch contributors functionality

### PR DESCRIPTION
**Pull request by niden**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 
 /vendor
 .env
+.gitignore
```
Comment (sergeyklay) [inline .gitignore]: @niden should git ignore the gitignore?

File: app/tasks/FetchContributorsTask.php [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 namespace Website\Cli\Tasks;
 
+use const APP_PATH;
```
Comment (sergeyklay) [inline app/tasks/FetchContributorsTask.php]: what is it? :)

File: .gitignore (hunk 1 above)
Comment (niden) [inline .gitignore]: Hmm gotta love PHPStorm. We can remove it it is irrelevant.

File: app/tasks/FetchContributorsTask.php (hunk 2 above)
Comment (niden) [inline app/tasks/FetchContributorsTask.php]: `APP_PATH` is the path of the application set in `index.php`. This use statement is the new "auto import" from PHP7 that speeds things up. Again PHPStorm does that automatically :)

**Pull request merged**